examples-experimental/potree/src/pointcloud-tileset.ts

```diff
@@ -1,5 +1,7 @@
 import {Vector3} from '@math.gl/core';
 import {DataSource} from '@loaders.gl/loader-utils';
+import {POTreeNode} from '@loaders.gl/potree';
+import {PotreeTraverser} from './potree-traverser';
```

Comment:
- This idea of having a special traverser for each format always seemed wrong to me - i.e. I was hoping that we would not need to copy this particular pattern from the current `TIleset3D` code
- The source should normalize the returned data so that a common traverser can handle the data.